src/pages/about.astro

```diff
@@ -0,0 +1,101 @@
+---
+import { z } from "zod";
+import { Image } from "astro:assets";
+import Layout from "../layouts/Layout.astro";
+import TeamMemberCard from "../components/team/TeamMemberCard.astro";
+import MemberDetails from "../data/team.json";
+import COPSLogo from "../assets/images/cops_logo.png";
+import AboutUs from "../assets/images/aboutus.svg";
+// Enforce the validation of the member data from the JSON
+const memberZod = z.object({
+  name: z.string().min(5).max(30),
+  role: z.enum(["maintainer", "core-team", "contributor"]),
+  description: z.string().min(10).max(100),
+  skills: z.string().array().min(1).max(5),
+  image: z.string(),
+  socials: z.object({
+    github: z.string().url(),
+    gmail: z.string().url().optional(),
+    linkedin: z.string().url().optional(),
+    twitter: z.string().url().optional(),
+    instagram: z.string().url().optional(),
+  }),
+  portfolio: z.string().url().optional(),
+});
+export type TeamMemberInterface = z.infer<typeof memberZod>;
+
+const membersZod = z.array(memberZod);
+membersZod.parse(MemberDetails);
+---
+
+<Layout title="Team">
+  <div class="flex flex-col text-xl gap-4">
+    <div
+      class="flex w-[100%] justify-center items-center text-white text-5xl font-bold"
+    >
+      About Us
+    </div>
+    <div class="text-white flex flex-col px-8 md:px-32">
+      <div class="flex flex-col md:flex-row gap-12">
+        <div class="md:w-[50%] flex flex-col gap-8 justify-center">
+          <div class="text-3xl font-semibold underline">
+            About LFX Mentorship Project
+          </div>
+          <div>
+            Welcome to the LFX Mentorship Project, a dynamic initiative born
+            from our collective commitment to advancing excellence in the realm
+            of Open Source. We, members of
+            <div class="flex items-center justify-center">
+              <Image class="h-[4rem] w-fit" src={COPSLogo} alt="" /> Club Of
```

Comment:
Do not use images in line with text. It is bad for accessibility.

Reply:
Should I remove the logo from that place and Just bold the COPS-IITBHU instead?

Reply:
How should I use Codepen for reference , please elaborate a little , and also about the paragraph styling , how would you like it to be, a bit more scattered or more condensed